### ISSUE TITLE
Update March referendums to show which ones were passed

### ISSUE THREAD

**Issue by anlawyer** [caciviclab/odca-jekyll#352]
Best case: set a "passed" flag somewhere in the spreadsheet to pull from (@sfdoran might need to do this).
Alternative: hard code it.

Note: See the 2018 election to see how we marked the candidates who were elected as "elected". Also note that we didn't show which referendums were passed in 2018.

**Comment by anlawyer** [comment]
Didn't they all pass? Do we still want to do this?

**Comment by elinaru** [comment]
Note - a "is approved" column exists already in the spreadsheet for ballot measures to be filled with "true" or "false"

**Comment by sfdoran** [comment]
Yes, let's do it! We have 10 measures on the 2022 ballot and some are controversial.

**Comment by sfdoran** [comment]
Show "Measure passed" if TRUE and "Measure failed" if FALSE. See candidate page for example.

**Comment by elinaru** [comment]
example - candidate page:  
<img width="1206" alt="Screen Shot 2022-09-04 at 12 49 34 PM" src="https://user-images.githubusercontent.com/6467815/188331064-55feb0a9-2f84-480d-81e2-1384553bc811.png">

**Comment by elinaru** [comment]
How about we style it more like a badge - like so: 
<img width="698" alt="Screen Shot 2022-09-04 at 12 58 01 PM" src="https://user-images.githubusercontent.com/6467815/188331335-b478872d-f77b-4aa6-850c-d923902005be.png">



    font-size: 2rem;
    color: white;
    background-color: #808080;
    padding: 2px 8px;
    border-radius: 6px;
